Tests/govuk_ios/govuk_ios_unit_tests/Specs/Extensions/Foundation/Date+ExtensionsTests.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import Testing
+
+@testable import govuk_ios
+
+@Suite
+struct Date_ExtensionsTests {
+    @Test
+    func isToday_today_returnsTrue() {
+        let sut = Date()
+        #expect(sut.isToday())
+    }
+
+    @Test(
+        arguments: [
+            "01/01/2001",
+            "02/12/2004",
+            "02/12/2024",
+            "02/10/2025"
```

Comment:
Do you want this date a bit farther in the future, so this doesn't fail next Feb?

Reply:
Yeah, I do. I am going to remove the date string and add a test that works out a future date on the fly